core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -257,16 +257,16 @@ private void runThreadLocalContextCustomizers() {
     }
 
     private void failEarly(Throwable cause) {
-        final RequestLogBuilder logBuilder = logBuilder();
-        final UnprocessedRequestException wrapped = new UnprocessedRequestException(cause);
-        logBuilder.endRequest(wrapped);
-        logBuilder.endResponse(wrapped);
-
         final HttpRequest req = request();
         if (req != null) {
             autoFillSchemeAndAuthority();
             req.abort(cause);
         }
+
+        final RequestLogBuilder logBuilder = logBuilder();
+        final UnprocessedRequestException wrapped = new UnprocessedRequestException(cause);
+        logBuilder.endRequest(wrapped);
+        logBuilder.endResponse(wrapped);
```

Comment:
This rearrangement is to make abort() is called before log completion, like we did in `ClientUtil.fail()`, for consistency.